sympy/solvers/solveset.py

```diff
@@ -811,6 +811,46 @@ def solveset_complex(f, symbol):
     return solveset(f, symbol, S.Complexes)
 
 
+def solvify(f, symbol, domain=S.Reals):
```

Comment:
I would prefer to be function which takes a set and returns the solution in a form which `solve` would return. So the usage would be `solvify(solveset(f, symbol, domain))`